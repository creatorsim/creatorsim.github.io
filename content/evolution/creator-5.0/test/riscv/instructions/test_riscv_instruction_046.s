
#
# Creator (https://creatorsim.github.io/creator/)
#

.text
main:

    li x5, 0x22346
    li x6, 4567
    li x7, -9486
    
    not x8, x5 
    not x9, x6 
  	not x10, x7
    