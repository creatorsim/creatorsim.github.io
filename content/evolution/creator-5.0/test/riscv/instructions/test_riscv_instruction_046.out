
cr[PC]:0x24; ir[x5,t0]:0x22346; ir[x6,t1]:0x11d7; ir[x7,t2]:0xffffdaf2; ir[x8,fp,s0]:0xfffddcb9; ir[x9,s1]:0xffffee28; ir[x10,a0]:0x250d; keyboard[0x0]:''; display[0x0]:''; 

